src/styles/components/_stars.scss

```diff
@@ -0,0 +1,25 @@
+.stars-list {
+  display: flex;
+}
+
+.stars-item {
+  width: 16px;
+  height: 16px;
+  background: url('../images/star.svg') center no-repeat;
+}
+
+.stars-list--active .stars-item:nth-child(-n + 4) {
+  background-image: url('../images/star-active.svg');
+}
+
+.card-stars_raiting {
+  font-size: 10px;
+  color: $mainColor;
```

Comment:
The variable `$mainColor` is used here, but it is not defined in this file. Ensure that `$mainColor` is defined in a file that is imported before this one, or define it in this file.